Configure.cmake

```diff
@@ -156,10 +165,21 @@ if(LD_SIZE GREATER "9")
   set(COMPILER_SUPPORTS_LONG_DOUBLE 1)
 endif()
 
+# Bug workaround for gcc 4.x
```

Comment:
Is this something you need on your machine? Or is this something that is happening just on the version of Ubuntu that Travis runs? I'd prefer `COMPILER_SUPPORTS_LONG_DOUBLE`  to be detected by the script as it currently does. What is preventing this detection to work?

Reply:
This is now replaced with a detection code.